variables.js

```diff
@@ -48,3 +56,6 @@ Steps:
         - Print the final message to the console, including the personName in uppercase in this format `Dear personName_VALUE, here's your message: finalMsg_VALUE.`.
 *******************************************************************************/
 // TODO: ADD YOUR CODE BELOW
+    let msg=prompt("enter a message:")
+    let finalMsg = `${msg} And btw, Why are you happy?`
+    console.log(`Dear ${personName.toUpperCase()}, ${finalMsg}`);
```

Comment:
Here you missed a part from the printed message which is: "here's your message:", the rest is perfect.